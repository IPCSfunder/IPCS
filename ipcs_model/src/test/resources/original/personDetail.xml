<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<person_detail person_detail_objid="1" last_name="Michael" first_Name="Jackson" age="35" sex="MALE" nationality="Chinese" dob="1987-08-25" nick_name="JJ" nric="8767656788" market_option="1"/>
</dataset>