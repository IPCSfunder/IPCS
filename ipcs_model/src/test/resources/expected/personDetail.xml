<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<person_detail last_name="Michael" first_Name="Jackson" age="35" sex="MALE" nationality="Chinese" dob="1987-08-25" nick_name="JJ" nric="8767656788" market_option="1"/>
	<person_detail last_name="Gard"  first_Name="Richard" age="25" sex="FEMALE" nationality="Singaporean" dob="1986-08-25" nick_name="GG" nric="1234567" market_option="0"/>
</dataset>